portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,4 +88,16 @@ private String convertToJsonUsingGson() {
     Gson gson = new Gson();
     return gson.toJson(comments);
   }
+
+  /**
+   * Converts an string to an integer.
+   */
+  private int toInt(String str) {
```

Comment:
This seems more specialized than strictly turning a string to an int. You should probably rename this to let anyone reading your code know this (and clarify in the comments as well). Something like 'numCommentsShownInputParseToInt' or something similar that gets that across.

Reply:
Ur right, fixed!